stdpopsim/slim_engine.py

```diff
@@ -934,8 +1087,9 @@ def matrix2str(
             # dealing with distributions given by "s" Eidos script,
             else:
                 distrib_args = "; ".join([f'"{a}"' for a in m.distribution_args])
-            mid = next_mutation_type_id
-            next_mutation_type_id += 1
+            # ensuring mutation type ids are unique
+            assert mid != last_mid
+            last_mid = mid
```

Comment:
We don't need to do this - if we were to check this somewhere, it should be in ` _dfe_to_mtypes`, but there it's totally obvious.
```suggestion
```

Reply:
yeah, I used this while debugging but it is not needed!